Makefile

```diff
@@ -12,7 +12,7 @@ RESET := $(shell tput -T linux sgr0)
 TITLE := $(BOLD)$(PURPLE)
 SUCCESS := $(BOLD)$(GREEN)
 # the quality gate lower threshold for unit test total % coverage (by function statements)
-COVERAGE_THRESHOLD := 45
+COVERAGE_THRESHOLD := 48
```

Comment:
is there a specific reason for this change?

Reply:
The current output from `make unit` is (from the latest run in the pipeline):
```
Coverage: 49.2
```
We try to bump the coverage threshold (if testing coverage falls below that threshold unit tests automatically fail) as we get more coverage (though this is a little close, but probably alright 👍 ).